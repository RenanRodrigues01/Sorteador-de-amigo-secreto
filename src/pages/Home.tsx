import Form from "../components/Form"
import List from "../components/List"
import Rodape from "../components/Rodape"

const Home = () => {
  return (
    <>
        <Form />
        <List />
        <Rodape />
    </>
  )
}

export default Home
