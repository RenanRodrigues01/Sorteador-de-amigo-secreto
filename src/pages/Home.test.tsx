import { render } from "@testing-library/react";
import React from "react";
import { RecoilRoot } from "recoil";
import Home from "./Home";

const mockNavegação = jest.fn();

jest.mock('react-router-dom', () => {
    return {
        useNavigate: () => mockNavegação
    }
} )
describe('A pagina inicial', () => {
    test('Deve renderizar corretamente', () => {
        const { container } = render(
            <RecoilRoot>
                <Home />
            </RecoilRoot>
        );
        
        expect(container).toMatchSnapshot()
    })
})