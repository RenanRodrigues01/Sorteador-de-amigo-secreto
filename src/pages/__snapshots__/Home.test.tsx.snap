// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A pagina inicial Deve renderizar corretamente 1`] = `
<div>
  <form>
    <input
      placeholder="Insira os nomes dos participantes"
      type="text"
      value=""
    />
    <button
      disabled=""
    >
      Adicionar
    </button>
    
  </form>
  <ul />
  <footer>
    <button
      disabled=""
    >
      Iniciar a Brincadeira
    </button>
  </footer>
</div>
`;
